peekingduck/training/src/trainer/tensorflow_trainer.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2023 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tensorflow trainer"""
+
+import logging
+from typing import Any, Dict, List, Optional, Union
+from omegaconf import DictConfig
+import tensorflow as tf
+from tqdm.keras import TqdmCallback  # for progress bar
+from configs import LOGGER_NAME
```

Comment:
To separate the above imports into two blocks: Python system imports and third party imports.

Reply:
https://github.com/furyhawk/PeekingDuck/tree/47df8145fbba97081c6758cf66856b571d37db99

Grouped.